tpm/tpm.go

```diff
@@ -628,6 +631,125 @@ func MakeIdentity(rw io.ReadWriter, srkAuth []byte, ownerAuth []byte, aikAuth []
 	return blob, nil
 }
 
+func unloadTrspiCred(blob []byte) ([]byte, error) {
+	/*
+	 * Trousers expects the asym blob to have an additional data in the header.
```

Comment:
This comment doesn't give me enough context about why this function is here. Naively, it seems like a layering violation for this code to have to take into account anything about the Trousers code above it.

Please expand the explanation. Does this mean, for example, that this library is now specific to Trousers?